dascore/io/rsf/core.py

```diff
@@ -79,7 +82,11 @@ def write(self, spool, path, data_path=None, **kwargs):
         hdr_info = [hdr_str, file_formt, f"esize={file_esize}"]
         for i in range(length):
             hdr_info.append(f"n{i+1}={axis_lengs[i]}")
-            hdr_info.append(f"o{i+1}={axis_origs[i]}")
+            if i == 0:
+                hdr_info.append(f"o{i+1}=0.0")
+                hdr_info.append(f"starttime={axis_origs[i]}")
+            else:
+                hdr_info.append(f"o{i+1}={axis_origs[i]}")
```

Comment:
I don't think we guarantee that the 0th axis is time here.  Maybe:

```python
if axis_names[i] == "time"
```
would be a better condition?